testing/maskfile.md

```diff
@@ -10,15 +10,8 @@ set -eEux
 
 echo "instanciating infra"
 
-kcli create host kvm -H 127.0.0.1 local
-
-kcli create plan -f "${SOFT_RESIDE}/infra/pools.yml" pools 
-kcli list pool
-kcli download image -p image_pool fedora30
-
```

Comment:
Also this was necessary, I iterated that out over several runs. I known it differs from the doc.